hack/ci-e2e-kind-upgrade.sh

```diff
@@ -194,9 +188,15 @@ make test-pre-upgrade GARDENER_PREVIOUS_RELEASE=$GARDENER_PREVIOUS_RELEASE GARDE
 
 echo "Upgrading gardener version '$GARDENER_PREVIOUS_RELEASE' to '$GARDENER_NEXT_RELEASE'"
 upgrade_to_next_release
-wait_until_seed_gets_upgraded "$SEED_NAME"
+
+echo "Wait until seed '$SEED_NAME' gets upgraded from version '$GARDENER_PREVIOUS_RELEASE' to '$GARDENER_NEXT_RELEASE'"
+kubectl wait seed $SEED_NAME --timeout=5m --for=jsonpath="{.status.gardener.version}=$GARDENER_NEXT_RELEASE"
+# RETRY_LIMIT has been increased to 240 (20 minutes) due to the upgrading of Gardener for seed.
```

Comment:
It would be better if the internal retry logic would not be exposed to the caller here. The caller wants to set a timeout, but has to calculate with the internal 5 second sleep delay to get the retry limit.
What do you think about changing it to a more relatable value like seconds or minutes?

Reply:
Thanks @ScheererJ for the suggestion.
I have fixed it so that the caller can now set the TIMEOUT in seconds. Operation will retry until the timeout is reached, with a 5-second sleep interval between each retry.